protocol/avdl/stellar1/local.avdl

```diff
@@ -35,7 +35,19 @@ protocol local {
   array<CurrencyLocal> getDisplayCurrenciesLocal(int sessionID);
 
   void changeWalletAccountNameLocal(int sessionID, AccountID accountID, string newName);
-  void deleteWalletAccountLocal(int sessionID, AccountID accountID);
+  
+
+  // This is theoretically, and most of the time - practically
+  // - irreversible operation. If user has not saved the secret key
+  // elsewhere, they will be unable to access or transfer out funds
+  // from the account after delete goes through. 
+
+  // There are two string arguments: `understoodKeybaseCantRecover`
+  // and `backedUpSecretKey`. This RPC is checking if both of them
+  // contain the string "yes". Consumer of this API should display a
+  // text edit prompt to the user explaining the above and pass the
+  // entered text to the RPC. all for additional user safety.
+  void deleteWalletAccountLocal(int sessionID, AccountID accountID, string understoodKeybaseCantRecover, string backedUpSecretKey);
```

Comment:
That's a bit much. I think people are more likely to read it if it's shorter.
Deleting an account is irreversible. The caller should prompt the user
`userAcknowledged bool` or `userConfirmation string`

Reply:
hah I assumed it's the separate in code docs - people will be intrigued about longer chunks of texts :) but I will make it shorter, it looks a bit out of place in this avdl file.